### DISCUSSION TITLE
Quick configuration for not filtering banner ads, text-ads or both

### DISCUSSION THREAD

**Discussion by joonas-foo** [uBlockOrigin/uBlock-issues#2485] [Ideas] [answered]
Hi! I had this idea that there would be quick configuration to not filter banner advertisements or text ads or both, this could be a global filter exception or just for selected webpages. Ads are sometimes important to keep a website online and I would want to see them on some webpages just for interest! :-)

**Comment by ghost** [comment, marked as answer]
Turn off cosmetic filtering. Or use ABP.

***

Make sure you know what "not blocking banner ads, text-ads" may cause - https://twitter.com/gorhill/status/1586412740989460481
